exploration/syntax-exploration-2.md

```diff
@@ -0,0 +1,184 @@
+# Syntax Exploration v2
+
+In the 2023-10-16 teleconference, we narrowed the options being considered for text-mode syntax to "1a", "2a", and "3a". 
+Each of these syntax candidates has been adjusted based on our conversation.
+
+One open issue, separately considered here, is the use of an additional sigil for some of the syntaxes.
+We have generally used `#` as the placeholder for this sigil in the previous iteration.
+
+
+## Comparison Matrix
+
+... goes here...
+
+## Candidates
+
+### Candidate 1a
+
+**Description:** Invert for Text Mode, distinguish statements from placeholders
+
+- Placeholders and expressions use `{`/`}`:
+  > `{$var}`, `{unquoted}`, `{|quoted|}`, `{$var :function}`
+- Non-placeholders use `{#`/`}`:
+  > `{#input $user}`, `{#local $var=$foo :expr}`, `{#match {$foo}}`, `{#when * *}`
+
+The use of `#` as the sigil is intentional here.
+This sigil might be replaced, but does not require additional escaping, as code is enclosed in `{`/`}`.
+Note that the `#` sigil might also be considered to be part of the keyword, e.g. `#when`, `#local` vs.
+allowing whitespace to appear between them.
+
+Patterns in this syntax are generally unquoted,
+but MAY be quoted with the 2023-10-09 consensus `{{`/`}}` quotes.
+
+ASCII whitespace trimming is assumed, but is a hot topic.
+
+**Examples**
+```
+Hello world!
+```
+```
+Hello {$user}
+```
+```
+{#input $user :person type=informal}
+Hello {$user}
+
+{#input $user :person type=informal}Hello {$user}
+```
+```
+{#input $var :function option=value}
+{#local $foo = $bar :function option=value}
+Hello {$var}, you have a {$foo}
+
+{#input $var :function option=value}{#local $foo = $bar :function option=value}Hello {$var}, you have a {$foo}
+```
+```
+{#match {$foo} {$bar}}
+{#when foo bar} Hello {$foo} you have a {$var}
+{#when * *} {$foo} hello you have a {$var}
+
+{#match {$foo} {$bar}}{#when foo bar}Hello {$foo} you have a {$var}{#when * *}{$foo} hello you have a {$var}
```

Comment:
It would be better to show here the one-selector case, as that's much more common.

```suggestion
{#match {$foo}}
{#when foo} Hello {$foo} you have a {$var}
{#when *} {$foo} hello you have a {$var}

{#match {$foo}}{#when foo}Hello {$foo} you have a {$var}{#when *}{$foo} hello you have a {$var}
```